src/rest/APIRequest.js

```diff
@@ -17,6 +17,9 @@ class APIRequest {
     this.options = options;
     this.retries = 0;
 
+    const userAgentSuffix = this.client.options.userAgentSuffix;
+    this.fullUserAgent = `${UserAgent}${userAgentSuffix.length && `, ${userAgentSuffix.join(", ")}`}`;
```

Comment:
```suggestion
    this.fullUserAgent = `${UserAgent}${userAgentSuffix.length ? `, ${userAgentSuffix.join(', ')}` : ''}`;
```

Reply:
I considered doing this. However, further down the line, I saw the same thing with `this.path`. Thought it would make the most sense if it were to be consistent.

Reply:
> I considered doing this. However, further down the line, I saw the same thing with `this.path`. Thought it would make the most sense if it were to be consistent.

It's a different implementation. For `queryString` to be falsy, it has to be an empty string. As for `userAgentSuffix.length`, it has to be 0, resulting in `${UserAgent}0`

Reply:
guess i forgot to add an actual comment, but yes pretty much what @almeidx said, in this case it would add a 0 after the user agent

Reply:
Ah, my bad! I didn't notice. Fix will be pushed soon™